src/Sulu/Bundle/AdminBundle/Resources/js/components/DatePicker/DatePicker.js

```diff
@@ -19,7 +19,11 @@ type Props = {|
     /** Listen for changes of this component */
     onChange: (value: ?Date) => void,
     /** Configure the datepicker to your needs, for more information have a look in the README.md */
-    options: Object,
+    options: {
+        [any]: any,
```

Comment:
Why is that necessary? I thought as long as you don't put the pipes in the object definition it should allow additional properties without this line.